### ISSUE TITLE
accessibility issue: unable to organise requests and folder

### ISSUE THREAD

**Issue by yongshing32** [getinsomnia/insomnia#9419]
### Expected Behavior

Screen reader user should able to organise the request-- move the requests into a folder and pull it out from a folder. 

### Actual Behavior

Currently, I can press tab key to reach the requests and collection table. I also can create new folder using the keystroke ctrl+shift+n. 
But, after folder created, I want to move requests in and out of the folder, there is no way to do so. 
The drag and drop method is not accessible by screen reader users because screen reader entirely rely on keyboard access. 

### Reproduction Steps

1. Open a collection. 
2. press tab key until screen reader reads 'request and collection table'
3. we can up down arrow to move through the items in the table.
4. I can see the folder and requests in the table/list. 
5. But, I want to move the requests into the folder, no way to accomplish this. 

### Is there an existing issue for this?

- [x] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Which sync method do you use?

- [ ] Git sync.
- [ ] Insomnia Cloud sync.
- [x] Local only

### Insomnia Version

Version: Insomnia 12.1.0 Build date: 11/21/2025 OS: Windows_NT x64 10.0.26200 Electron: 38.4.0 Node: 22.20.0 Node ABI: 139 V8: 14.0.365.10-electron.0 Architecture: x64

### What operating system are you using?

Windows

### Operating System Version

Windows 11 pro 25H2 (OS Build 26200.7171)

### Installation method

download from official site

**Comment by ThunderDK** [comment]
Is it the same issue I have where I can't very easily drag a folder into a different "height" or even the same "height" as other folders? Basically the organizational part of the folders are not so easy.
